run-local-dynamodb-titan.sh

```diff
@@ -0,0 +1,7 @@
+mvn install
+echo 'compiled dynamodb-titan-storage-backend using maven'
+mvn test -Pstart-dynamodb-local &
+echo 'started dyanmodb local'
+cd server/dynamodb-titan100-storage-backend-1.0.0-hadoop1
```

Comment:
This directory only exists if you run the src/test/resources/install-gremlin-server.sh script.
Missing from your script.
Instead of building from scratch, why don't you download the publicly served zip (linked [here](http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Tools.TitanDB.GremlinServerEC2.html)):
https://s3.amazonaws.com/dynamodb-titan-us-east-1/dynamodb-titan100-storage-backend-1.0.0-hadoop1.zip
and add an option to build the server directory from scratch if desired. The HTML link contains the SHA1 hash of the zip, so you can skip downloading the file if the SHA1 hash on the web page matches the hash of the zip already in the directory.